routemaster/state_machine/utils.py

```diff
@@ -184,7 +188,9 @@ def _labels_in_state(
         ranked_transitions.c.label_name,
     )).where(and_(*(active_filters + tuple(filters))))
 
-    return [x for x, in conn.execute(active_participants)]
+    y = [x for x, in conn.execute(active_participants)]
+    print(y)
```

Comment:
`print` -> logging.

Should we add https://pypi.python.org/pypi/flake8-print ?

Reply:
Yep I forgot about this one. Added in ebd749ea308e99a744fb2b3d35c24bfcc316d2b7